modules/bookshelf/README.md

```diff
@@ -0,0 +1,14 @@
+# 📚 Bookshelf Library
```

Comment:
For consistency also if we have bookshelf-dev and bookshelf-<whatever>, we may need one. This way readmes for release are a bit simplier since i dont think we need a full readme on platforms. A shorter description with links is better imo

Reply:
We can keep that for the moment and will discuss it before the release.